libs/ui-components/src/lib/TodoList/TaskComponents/TaskModal.tsx

```diff
@@ -85,131 +85,132 @@ export const TaskModal: React.FC<TaskModalProps> = ({
+            <>
+              <ModalHeader>
+                <TableContainer fontSize="md">
+                  <Table variant="simple">
+                    <Thead>
+                      <Tr>
+                        <Th>Category:</Th>
+                        <Th>{task.category.name}</Th>
+                      </Tr>
+                    </Thead>
+                    <Tbody>
+                      <Tr>
+                        <Td>Name:</Td>
+                        <Td>{task.name}</Td>
+                      </Tr>
                       <Tr>
-                        <Td>Deadline:</Td>
-                        <Td>
-                          {deadlineToDate(
-                            task.chunkInfo.deadline,
-                            DESCRIPTIVE_DATE_FORMAT
-                          )}
-                        </Td>
+                        <Td>Priority:</Td>
+                        <Td>{task.priority}</Td>
                       </Tr>
-                    )}
-                    {!task.isFloat && task?.chunks && task.chunks.length > 0 && (
-                      <>
+                      {task.isFloat && task?.chunkInfo?.deadline && (
                         <Tr>
-                          <Td>Date:</Td>
+                          <Td>Deadline:</Td>
                           <Td>
-                            {momentToDate(
-                              task.chunks[0].start,
+                            {deadlineToDate(
+                              task.chunkInfo.deadline,
                               DESCRIPTIVE_DATE_FORMAT
                             )}
                           </Td>
                         </Tr>
-                        <Tr>
-                          <Td>Time:</Td>
-                          <Td>{startToTime(task.chunks[0].start)}</Td>
-                        </Tr>
-                      </>
-                    )}
-                  </Tbody>
-                </Table>
-              </TableContainer>
-            </ModalHeader>
-            <ModalCloseButton />
-            <ModalBody>
-              {task.isFloat && (
-                <Box>
-                  <TaskChunks chunks={task.chunks!} />
-                </Box>
-              )}
-            </ModalBody>
+                      )}
+                      {!task.isFloat && task?.chunks && task.chunks.length > 0 && (
+                        <>
+                          <Tr>
+                            <Td>Date:</Td>
+                            <Td>
+                              {momentToDate(
+                                task.chunks[0].start,
+                                DESCRIPTIVE_DATE_FORMAT
+                              )}
+                            </Td>
+                          </Tr>
+                          <Tr>
+                            <Td>Time:</Td>
+                            <Td>{startToTime(task.chunks[0].start)}</Td>
+                          </Tr>
+                        </>
+                      )}
```

Comment:
We've fixed this part in my PR with @karpinsk.
So we need to merge that in order of PRs openings, my first, than this.

Reply:
rollbacked commit regarding taskmodal fix - merging since now it's related only to google button